streamwebs_frontend/streamwebs/views.py

```diff
@@ -85,7 +49,6 @@ def register(request):
             profile.user = user
             profile.save()
             registered = True
-
         else:
```

Comment:
@leian7 Thanks, that really cleans up the test output.  Can I comment out (or remove) those lines in this PR?